apache/zeppelin
# [ZEPPELIN-4987]. Fix the interpreter dependency conflict caused by the CLASSPATH environment variable

### What is this PR for?
Fix the interpreter dependency conflict caused by the CLASSPATH environment variable

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
* [ZEPPELIN-4987](https://issues.apache.org/jira/browse/ZEPPELIN-4987)

### How should this be tested?
* First, Export the CLASSPATH environment variable in /etc/profile or ~/.bashrc.
* Then, Remove the CLASSPATH configuration in zeppelin-env.sh
* Finally, Restart Zeppelin Server and start Spark Interpreter，Check whether the zeppelin lib directory is included in the interpreter process CLASSPATH, if not, the test passes.
 
### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
